crates/astria-sequencer/src/state_ext.rs

```diff
@@ -244,18 +269,56 @@ pub(crate) trait StateWriteExt: StateWrite {
 
 impl<T: StateWrite> StateWriteExt for T {}
 
+fn revision_number_from_chain_id(chain_id: &str) -> u64 {
```

Comment:
However unlikely, I think this could do with a `anyhow::Result<Option<u64>>` instead of panicking.

For the error case (u64 overflow): `init_chain` allows returning an error, so returning it is not a big deal.

For `Option::None`: the code should emit a event that no revision was found and is hence assumed 0

Reply:
init_chain erroring is effectively the same as panicking as the node will shut down and not be able to initialize. will add a log for the no revision number case tho